### PR TITLE
added scene handling documentation

### PR DESCRIPTION
**Pull request by cdingm**
i added a explanation on how we currently handle scenes in unity 
@marlau97 maybe you want to add a chapter to this document on how the scene handling is done when entering a dungeon end exiting it? I believe it would be nice to have

### CODE REVIEW

File: dev-manuals/languages/unity/scene-handling.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Scene handling
+
+Here it is documented how we currently handle multiple scenes in Unity.
+
+## Requirements
+
+To use our unity scenes with script code we need to add them to the Build Settings.  
+![build settings](build-settings-scenes.png)  
+Simply just drag and drop the required Scenes into the "Scenes in Build" area.
```
Comment (delvh) [inline dev-manuals/languages/unity/scene-handling.md:9]:
```suggestion
Simply just drag and drop the required Scenes into the `Scenes in Build` area.
```

File: dev-manuals/languages/unity/scene-handling.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+Every Scene that needs to be in the Final Version, has to be there.
+
+## First Scene
+
+We need to have a starting Scene from which every other Scene is accessible in some way. Currently we have a dummy scene called "FirstScene" which has a Script attached to it. If you load the "FirstScene" it executes a Script "LoadFirstScene.cs" that is attached to it. This Script loads the configured Scenes.
```
Comment (delvh) [inline dev-manuals/languages/unity/scene-handling.md:14]:
```suggestion
We need to have a starting Scene from which every other Scene is accessible in some way. Currently we have a dummy scene called `FirstScene` which has a Script attached to it. If you load the `FirstScene` it executes a Script `LoadFirstScene.cs` that is attached to it. This Script loads the configured Scenes.
```

Comment (delvh) [reply dev-manuals/languages/unity/scene-handling.md:14]: That script should probably be renamed to `LoadScenes.cs`...

File: dev-manuals/languages/unity/scene-handling.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    void Update()
+    {
+        
+    }
+}
+
+
+```
+
+In this Example the Scene "Area 1" is loaded and the HUD Scene is also loaded additively.
```
Comment (delvh) [inline dev-manuals/languages/unity/scene-handling.md:46]:
```suggestion
In this example, the Scene `Area 1` is loaded, and the HUD Scene is also loaded additively.  
```
Should also probably be ~~Area~~ `World 1` now.

File: dev-manuals/languages/unity/scene-handling.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    {
+        
+    }
+}
+
+
+```
+
+In this Example the Scene "Area 1" is loaded and the HUD Scene is also loaded additively.  
+There also is a line which specifies the starting Point of the Player in the loaded Scene.
```
Comment (delvh) [inline dev-manuals/languages/unity/scene-handling.md:47]: There is 
```suggestion
There is also a line that specifies the starting point of the Player in the loaded Scene.  
```

**Review by delvh — APPROVED** [review]

**Review by delvh — APPROVED** [review]

**Review by marlau97 — APPROVED** [review]

**Pull request merged**